src/pages/news/news-detail/news-detail.js

```diff
@@ -115,4 +118,14 @@ const NewsDetailPage = ({ match }) => {
   );
 };
 
+const newsDetailPropTypes = {
```

Comment:
There is no big reason to create a separate variables for props except cases where you are going to reuse it somewhere. You can just use `NewsDetailPage.propTypes = {...}`

Reply:
redone